meson_options.txt

```diff
@@ -0,0 +1,15 @@
+# meson options for open-isns
+
+# enabled security and service location protocol usage by default
+option('security', type: 'feature', value: 'enabled')
+option('slp', type: 'feature', value: 'enabled')
+
+# disable shared library creation and enable static library
```

Comment:
If you use default_library instead, then you can just add `'default_library=static'` to the project default options.